bin/prometheus_exporter

```diff
@@ -34,6 +35,9 @@ def run
     opt.on('--prefix METRIC_PREFIX', "Prefix to apply to all metrics (default: #{PrometheusExporter::DEFAULT_PREFIX})") do |o|
       options[:prefix] = o.to_s
     end
+    opt.on('--label METRIC_LABEL', "Label to apply to all metrics (default: #{PrometheusExporter::DEFAULT_LABEL})") do |o|
+      options[:label] = JSON.parse(o.to_s)
```

Comment:
Do we need a full JSON.parse here?

Reply:
`bin/prometheus_exporter --label '{"environment": "integration"}'`

Because a label requires a hash, that's why it requires a JSON.parse. Unless we breakdown the key and value in different arguments. I didn't choose that method because it will be harder to specify multiple labels.

`bin/prometheus_exporter --label '{"environment": "integration", "test": "valueA"}'`
VS
`bin/prometheus_exporter --label-key environment,test --label-value integration,valueA`

Reply:
@SamSaffron any feedback?